app/assets/javascripts/discourse/controllers/bookmark.js

```diff
@@ -413,6 +439,22 @@ export default Controller.extend(ModalFunctionality, {
         .finally(() => (this.saving = false));
     },
 
+    delete() {
+      this.deleting = true;
+      bootbox.confirm(I18n.t("bookmarks.confirm_delete"), result => {
+        if (result) {
+          this.closeWithoutSaving = true;
```

Comment:
I know this variable didn't start in this commit but as a general note - if you are only using a variable in a particular controller, and nothing is computed based on it, we prefer to underscore them as private. For example `this._closeWithoutSaving`. 

Absolutely a minor thing, though and just something to keep in mind for the future.

Reply:
@eviltrout cool I was talking to Joffrey about this as well RE: functions because I wasn't sure, so I will make sure to do it from now on :)